File: Python_2020.2/Unidade1/Algoritmos-Lista_2/Exerc_2.py
#Faça um programa para a leitura de duas notas parciais de um aluno. O programa deve calcular a média alcançada por aluno e apresentar:
#A mensagem "Aprovado", se a média alcançada for maior ou igual a sete;
#A mensagem “Prova Final”, se a média alcançada estiver entre quatro e sete;
#A mensagem "Reprovado", se a média for menor do que quatro.

nota1 = float(input('digite sua nota1:'))
nota2 = float(input('digite sua nota2:'))
media = (nota1 + nota2) / 2

if media >= 7:
  print('aprovado')
elif media >=4: 
  print('prova final')
else:
  print('reprovado')  

File: LinguagemC_2021.1/Unidade1/Programacao1-Lista_1/Exerc_2.c
/*
2. Considere uma disciplina que adota o seguinte critério de aprovação: os alunos fazem duas provas (P1 e P2) iniciais; se a média nessas duas provas for maior ou igual a 5.0, e se nenhuma das duas notas for inferior a 3.0, o aluno passa direto. Caso contrário, o aluno faz uma terceira prova (P3) e a média é calculada considerando-se essa terceira nota e a maior das notas entre P1 e P2. Neste caso, o aluno é aprovado se a média final for maior ou igual a 5.0. Escreva um programa que leia inicialmente as duas notas de um aluno, fornecidas pelo usuário via teclado. Se as notas não forem suficientes para o aluno passar direto, o programa deve capturar a nota da terceira prova, também fornecida via o teclado. Como saída, o programa deve imprimir a média final do aluno, seguida da mensagem "Aprovado" ou "Reprovado", conforme o critério descrito acima. 
*/

#include <stdio.h>

int main(void) {
  int nota1, nota2, media, nota3;
  printf("\nDigite a sua primeira nota e segunda nota: ");
  scanf("%d %d", &nota1, &nota2);
  media = (nota1 + nota2) / 2;
  if (media >= 5 && nota1 >= 3 && nota2 >= 3) {
    printf("Aprovado!");
  } 
  else {
    printf("\nDigite a terceira nota: ");
    scanf("%d", &nota3);
    if (nota1 > nota2) {
      media = (nota1 + nota3) / 2;
    }
    else {
      media = (nota2 + nota3) / 2;
    }
      if (media >= 5) {
        printf("Sua média é %d e você foi Aprovado!", media);
    } else {
        printf("Sua média é %d e você foi Reprovado!", media);
    }
  }
   
  return 0;
}